src/lib/components/index/hero.svelte

```diff
@@ -2,16 +2,52 @@
   import Link from '$lib/components/index/link.svelte';
 </script>
 
+<head>
+  <link rel="stylesheet" href="css/index.css" />
+</head>
+
 <div class="space-y-8">
   <h1
-    class="mx-auto bg-gradient-to-r from-primary-100 via-accent-magenta to-accent-cyan bg-clip-text text-center text-3xl font-bold text-transparent dark:to-primary-400 md:text-6xl"
+    class="gradient-flow webkit-bg-clip-text mx-auto inline-block max-w-3xl bg-gradient-to-r
+   from-primary-100 via-accent-magenta to-accent-cyan bg-clip-text text-center text-3xl font-bold text-transparent md:text-6xl"
   >
     Good-first-issue-finder
   </h1>
+
   <h2 class="max-w-2xl text-center md:text-xl">
     Your entry point for finding good first issues in the EddieHub Organization and in the community
   </h2>
   <div class="flex justify-center">
     <Link variant="primary" href="/app">Get started</Link>
   </div>
 </div>
+
+<style lang="postcss">
+  .gradient-flow {
+    background-size: 200% 100%;
+    animation: gradientAnimation 2s linear infinite alternate-reverse;
+  }
+
+  @media (prefers-reduced-motion: reduce) {
```

Comment:
We don't need a dedicated media-query here, tailwind also provides them out-of-the-box with `reduced-motion:` [see their docs](https://tailwindcss.com/docs/animation#prefers-reduced-motion)